include/edyn/edyn.hpp

```diff
@@ -238,7 +238,8 @@ void set_should_collide(entt::registry &registry, should_collide_func_t func);
  */
 template<typename... Component>
 void refresh(entt::registry &registry, entt::entity entity) {
-    if (auto *coordinator = registry.try_ctx<island_coordinator>(); coordinator) {
+    auto* coordinator = registry.ctx().find<island_coordinator>();
```

Comment:
In the code style being used, `*` and `&` are placed before the name, not after the type, akin to EnTT's code style.